Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM python:3
 WORKDIR /work
-COPY requirements.txt /work
```

Comment:
I'm not even sure if this is the right solution at all, but if there is a "COPY ." at all, it shouldn't occur until after the pip install as this way around means it'll reinstall everything every time anyone makes any source changes as opposed to only when the dependencies change.